### DISCUSSION TITLE
关于在Windows端增加文件接受完成后有提示音和通知的建议

### DISCUSSION THREAD

**Discussion by ClearBrightFuture** [localsend/localsend#693] [L-简体中文]
-提示音提示文件已传输完成。

-在传输完成后增加一个一键跳转至该文件所在文件夹的按钮。这个按钮最好能集成在通知弹窗里。

-可能有的人不喜欢有提示音和通知弹窗，也建议在设置里增加开关按钮来让用户自己选择是否使用这项功能。

**Comment by dox-xu** [comment]
一年了，还是没有加
